test/Microsoft.AspNet.Mvc.FunctionalTests/ViewEngineTests.cs

```diff
@@ -288,7 +292,7 @@ public static IEnumerable<object[]> RazorViewEngine_UsesExpandersForLayoutsData
 View With Layout
 </language-layout>";
 
-                yield return new[] { "gb", expected1 };
+                yield return new[] { "en-GB", expected1 };
```

Comment:
Yes. `CultureInfo` does not recognize `gb`. So changed it to valid culture name.